crates/router/src/connector/aci.rs

```diff
@@ -50,6 +50,29 @@ impl ConnectorCommon for Aci {
             auth.api_key.into_masked(),
         )])
     }
+
+    fn build_error_response(
+        &self,
+        res: types::Response,
+    ) -> CustomResult<types::ErrorResponse, errors::ConnectorError> {
+        let response: aci::AciErrorResponse = res
+            .response
+            .parse_struct("AciErrorResponse")
+            .change_context(errors::ConnectorError::ResponseDeserializationFailed)?;
+        Ok(types::ErrorResponse {
+            status_code: res.status_code,
+            code: response.result.code,
+            message: response.result.description,
+            reason: response.result.parameter_errors.and_then(|errors| {
+                errors.first().map(|error_description| {
```

Comment:
here, connector always have only one element or more than one?? if more than one element, can we concatenate all the errors and pass it in reason?

Reply:
Done👍